rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_reader :person, :book, :date
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+
+    book.rentals.push(self)
+    person.rentals.push(self)
+  end
+
+  def book=(book)
+    @book = book
+    book.rentals.push(self) unless book.rentals.include?(self)
+  end
+
+  def person=(person)
```

Comment:
- [ ] kindly remove the `=` after the name of the method. use ` def person(person)` instead of    `person=(person)`